README.md

```diff
@@ -272,6 +347,31 @@ For cluster to get properly authorized it is expected that firewall is already c
       roles:
          - { role: 'ondrejhome.ha-cluster-pacemaker', cluster_name: 'vmware-cluster', cluster_configure_fence_xvm: false, cluster_configure_fence_vmware_rest: true, cluster_configure_stonith_style: 'one-device-per-cluster' }
 
+**Example playbook Resources configuration** .
+
+    - hosts: cluster
+      vars:
+        cluster_property:
+          - name: 'stonith-enabled'
+            value: 'false'
+          - name: 'no-quorum-policy'
+            value: 'ignore'
```

Comment:
Would it be possible to provide less controversial cluster properties here? 

The `stonith-enabled=false` and `no-quorum-policy=ignore` would render this cluster unsupportable on RHEL.
As a better example I would suggest `maintenance-mode=true` from the `pcs_property` examples doc that will create cluster in maintanence mode.

Reply:
Yes sure! I juste pickup one of my test. It should changed now

Reply:
> Hello @OliPou,
> 
> Thank you for using the ha-cluster-pacemaker role and for opening the PR to include support for configuring additional aspects of cluster such as resources and properties.
> 
> In additional to few comments/requests in code changes I have the following questions to this PR:
> 
>     * On which system have you tested the changes? (Debian 10 only or also some other systems)
> 
>     * Which Ansible version you have used to test changes here? I typically run tests on versions 2.8, 2.9 and 2.10 these days)
> 
>     * What is the reason for changes in `README.md` on lines 75, 81, 89 and 116? It looks to me that it was most probably caused by editor trying to automatically "fix something".
> 
> 
> Ondrej (온드레이)

Hi Ondrej,

Thanks for this really quick reply.
To answer to your questions: 
- I 'm afraid I have been only able to test it on Debian 10
- I was using ansible 2.10 version but I'm more than happy to test on this other versions.
- Yes sorry it seems that those lines included some blank space that were automatically removed by my editor, should I put them back?

Reply:
this change looks OK, resolving